diagnostic.md

```diff
@@ -177,5 +226,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+At 550px, an element with the class '.stretch' will be 30%. Any width less than 800px and creater than 400px will trigger the media query for a width of 30%. 550px * 30% = 165px.
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have .stretch be 25% and at 550px width, 25% is 137.5 px.